src/types.ts

```diff
@@ -21,9 +21,12 @@ export type ProjectConstants = {
 };
 
 export type TokenConstants = {
```

Comment:
if someone is using surge env to deploy,  ChainSlug enum may not have that chain, therefore constants file will look like {9887:{}} instead of  {[ChainSlug.NEW_CHAIN]:{}}.  keeping type as ChainSlug below was giving typescript errors.

Reply:
put this as a code comment instead of here